Docker/testout/.env

```
@@ -0,0 +1,2 @@
+SERVER=testout-server
+GIT_URL=https://github.com/3CS-Software/test-out.git
```

When I tried this at first it didn't work because this repo uses main, and docker defaults to master :(
If we just explicitly put the branch in the url, it resolves this

```suggestion
GIT_URL=https://github.com/3CS-Software/test-out.git#main
```